### ISSUE TITLE
uploaded expense receipts get corrupted

### ISSUE THREAD

**Issue by savishy** [sapplica/sentrifugo#182]
Hello, 

We are finding that recently, the following issue occurs consistently:

### Steps to Reproduce

* I add receipts under `Expenses > Add > Receipts`
* I upload a file (PDF or Zip) using the "Click here to attach" file-picker. 
* The file *seems* to be uploaded correctly, although I see no progress bar of any sort. 
* I then select the uploaded file and download it using the red "Download" button.
* The file is downloaded to my machine. 

When I open this downloaded file, I find that it has been corrupted, even though the original uploaded file was proper in every way. 

This is blocking us from being able to process expenses seamlessly; could I request some assistance, or pointers on what might be the problem? 

Thanks!

**Comment by sapplica** [comment]
Hi,

Kindly check whether the uploaded file is present in 'expenses_receipts' file mentioned in the below path. If yes, open the file and confirm us whether the data is correct or not. Thank you!

File path:  \...(application location...)\...(application name)...\public\uploads\expenses_receipts

**Comment by Vakil12** [comment]
yes, the file is there but after the download file is corrupted.
